api/cloud/oc_cloud_manager.c

```diff
@@ -665,11 +697,17 @@ cloud_manager_login_async(void *data)
   return OC_EVENT_DONE;
 
 retry:
+  if (!cloud_schedule_retry(ctx, OC_CLOUD_ACTION_LOGIN,
+                            cloud_manager_login_async,
+                            OC_STATUS_SERVICE_UNAVAILABLE)) {
+    OC_CLOUD_ERR("Cannot schedule retry login, restarting");
+    cloud_set_cps_and_last_error(ctx, OC_CPS_FAILED, CLOUD_ERROR_CONNECT);
+    ctx->store.status |= OC_CLOUD_FAILURE;
+    oc_reset_delayed_callback(ctx, cloud_manager_reconnect_async, 2);
```

Comment:
why the 2 second delay for cloud_manager_reconnect_async, it usually has 0 ...?

Reply:
changed to 0